app/src/main/java/chat/rocket/android/app/DateTimeHelper.kt

```diff
@@ -36,18 +34,33 @@ object DateTimeHelper {
     }
 
     /**
-     * Returns a time from a LocalDateTime.
+     * Returns a time from a [LocalDateTime].
      *
-     * @param localDateTime The LocalDateTime.
-     * @return The time from a LocalDateTime.
+     * @param localDateTime The [LocalDateTime].
+     * @return The time from a [LocalDateTime].
      */
     fun getTime(localDateTime: LocalDateTime): String {
         val formatter = DateTimeFormatter.ofLocalizedTime(FormatStyle.SHORT)
         return localDateTime.toLocalTime().format(formatter).toString()
     }
 
-    private fun formatDate(localDate: LocalDate): String {
+    /**
+     * Returns a [LocalDateTime] from a [Long].
+     *
+     * @param long The [Long]
+     * @return The [LocalDateTime] from a [Long].
+     */
+    fun getLocalDateTime(long: Long): LocalDateTime {
```

Comment:
Why not use extension functions instead of Helper classes?
`fun Long.toLocalDateTime(): LocalDateTime`

`val date = timestamp.toLocalDateTime()`

We get a more idiomatic code and less Helper and Utils classes.

Reply:
Keep an eye on my comment below... The same for it...